src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/PFD/CJ4_PFD.js

```diff
@@ -877,7 +877,22 @@ class CJ4_APDisplay extends NavSystemElement {
         const apMasterActive = SimVar.GetSimVarValue("AUTOPILOT MASTER", "Bool") == 1;
         const ydActive = SimVar.GetSimVarValue("AUTOPILOT YAW DAMPER", "Boolean") == 1;
         const flightDirector = SimVar.GetSimVarValue("AUTOPILOT FLIGHT DIRECTOR ACTIVE", "Boolean") == 1;
-        
+        const precisionApproachMode = false //TODO set to match description in manual (enables double arrow in FMA)
+        const rightFDEngaged = false //TODO set when right FD is primary (turns arrow in FMA)
+
+        // TODO work in basic implementation of arrow logic
```

Comment:
@cwburnett I don't know how you'd most efficiently determine those states, so I gave you some uncommented examples of how the css classes work